code/datums/hud/ai.dm

```diff
@@ -62,6 +63,9 @@
 		viewport = create_screen("viewport", "Create Viewport", 'icons/mob/hud_ai.dmi', "viewport", "WEST, NORTH-2.5", HUD_LAYER)
 		viewport.underlays += "button"
 
+		hologram = create_screen("hologram", "Create Hologram", 'icons/mob/hud_ai.dmi', "hologram", "WEST, NORTH-3", HUD_LAYER)
```

Comment:
unless I'm mistaken `HUD_LAYER` as with the rest of the layers have been deprecated in favor of planes, `PLANE_HUD`

Reply:
Idk, who said that? I've still seen it used and it does need a layer. The rest of the file uses HUD_LAYER as well.

Reply:
than im probably mistaken haha. The person that I believe did the refactor is unfortunately not around for me to ask